components/common/src/ome/api/IContainer.java

```diff
@@ -32,8 +30,8 @@
  * The names of the methods correlate to how the function operates:
  * <ul>
  * <li><b>load</b>: start at container objects and work down toward the
- * leaves, returning hierarchy (Proejct-&gt;Dataset-&gt;Image</li>
- * <li><b>find</b>: start at leave objects and work up to containers,
+ * leaves, returning hierarchy (Project-&gt;Dataset-&gt;Image</li>
+ * <li><b>find</b>: start at leaf objects and work up to containers,
```

Comment:
Agreed, I didn't originally realize how hard we were avoiding branching `dev_5_2`.